internal/graph/model/signal_aggs.go

```diff
@@ -0,0 +1,23 @@
+package model
+
+import "time"
+
+type AliasKey struct {
+	Name string
+	Agg  string
+}
+
+type SignalAggregations struct {
+	Timestamp time.Time `json:"timestamp"`
+
+	ValueNumbers map[AliasKey]float64 `json:"-"`
+	ValueStrings map[AliasKey]string  `json:"-"`
+}
+
+type AggSignal struct {
```

Comment:
Maybe this shouldn't be here.

Reply:
It seems appropriate to me. This is just a generic model package for the repo, and this is related to the signal aggregation query.